adops/price-granularity.md

```diff
@@ -0,0 +1,178 @@
+| Any bid > 20.00 | 20.00 | n/a |
+| | | Total: 111 |
+
+
+#### Dense
+
+Dense provides a sliding scale similar to auto, but with smaller granularity. Any bid over the cap falls into the next highest bucket.
+
+{: .table .table-bordered .table-striped }  
+| Increment | Cap | Number of Line Items Required per Bidder |
+| 0.01 | 3.00 | 301 |
+| 0.05 | 8.00 | 100 |
+| 0.50 | 20.00 | 24 |
+| Any bid over 20.00 | 20.00 | n/a |
+| | | Total: 425 |
+
+
+#### Custom
+
+Custom buckets allow you to set your own price granularity. In determining what that granularity should be, we recommend analyzing your average bid prices from SSPs and placing them into a histogram. The price cap should be around your 95th-percentile bid, and there should be finer-grained buckets where more popular bid values are clustered. Note that if you use a [price floor](/dev-docs/modules/floors.html#price-floors-module), you shouldn't need fine-grained buckets below that floor.
+
+After you’ve determined what you’d like your granularity and caps to be, work with your software engineers to ensure the Prebid configuration matches your line item setup.
+
+## Pros and Cons
+
+As we mentioned earlier, you need to create a line item for every price that could come in from your header bidding demand partners. Without price buckets this could result in an almost infinite number of line items. Instead, you’ll need to weigh the pros and cons of high vs low granularity, and possibly adjust your granularities as you analyze the results of your Prebid implementation.
+
+The following examples give a general idea of the pros and cons of high and low granularity.
+
+### Example: High Price Granularity
+
+-  10 bidders
+-  Cap 20.00
+-  Increment .01
+-  Send top price: Minimum 2,000 line items
+-  Send all bids: Minimum 20,000 line items
+
+**Pro**: At .01 increments, there will be minimal price rounding (up to the price cap). All bids will compete at (or very near) the price the demand partner is willing to pay, maximizing revenue.
+
+**Con**: You have to create a lot of line items. The numbers shown above increase dramatically as you increase your price cap and your number of bidders. If you send all bids and add five more bidders, suddenly you need 10,000 more line items.
+
+### Example: Low Price Granularity
+
+-  10 bidders
+-  Cap 5.00
+-  Increment 1.00
+-  Send top price: Minimum 5 line items
+-  Send all bids: Minimum 50 line items
+
+**Pro**: Setting up your line items is a relatively quick process because you won’t need very many. You’re also less likely to run into volume limits your ad server might impose on numbers of line items.
+
+**Con**: Prices received from demand partners will be rounded down, so you could be losing money. A bid of 2.95 would be rounded down to 2.00, and lose to an ad server bid of 2.05. You’d be losing almost a dollar CPM.
+
+The following diagram, based on the scenarios we just described, illustrates just how quickly your line item count can grow.
```

Comment:
This is a great diagram (kittens!), but might be worth putting the assumptions as a footnote. e.g. "(Assumes 10 bidders)"